src/it/scala/net/snowflake/spark/snowflake/VariantTypeSuite.scala

```diff
@@ -4,6 +4,8 @@ import org.apache.spark.sql.types._
 import net.snowflake.spark.snowflake.Utils.SNOWFLAKE_SOURCE_NAME
 import org.apache.spark.sql.{Row, SaveMode}
 
+import scala.util.parsing.json.JSON
```

Comment:
Can we move this to ResultSetSuite?

Reply:
Do you mean to merge this test case to new test file: SnowflakeResultSetRDDSuite.scala? I think it may better to keep the test case to be separate. According test case name, it mainly test VARIANT type. It is very clear. SnowflakeResultSetRDDSuite is targeted to test new cases. This test case is updated because the data return format is some different but logically correct in the new data reading mode.